docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -160,7 +204,9 @@ A list of outputs (i.e. forwarding targets) for the Elastic Serverless Forwarder
 
 `inputs.[].outputs.[].type`:
 
-The type of the forwarding target output (currently only `elasticsearch` supported).
+The type of the forwarding target output (currently only `elasticsearch` and `logstash` are supported).
+
+NOTE: `logstash` target output is currently offered as Technology Preview.**
```

Comment:
Do you mean to stay technical preview here? Here is the boiler plate text that gets added when you use the [technical preview admonition](https://github.com/elastic/docs/blob/master/README.asciidoc#using-the-technical-preview-admonition) in the docs: 

![image](https://user-images.githubusercontent.com/14206422/216503365-4bbf8340-22b0-4249-9cef-553c5645b6e9.png)

Is technology preview different of the same? If the same, we should add text that matches.